guides/TrainingModelsHPC.md

```diff
@@ -92,76 +92,116 @@
+   - `learning_rate`: The lower bound and the upper bound of the learning rate parameter to consider.
+   - `n_epochs`: The lower bound and the upper bound of the number of epochs to consider.
+
+   To run Optuna, we can add `--optuna` arguments to the bash script. For example:
+
+   ```
+   train-detector  \
+   --dataset_dirs $DATASET_DIR \
+   --config_file $TRAIN_CONFIG_FILE \
+   --accelerator gpu \
+   --experiment_name $EXPERIMENT_NAME \
+   --seed_n $SPLIT_SEED \
+   --mlflow_folder $MLFLOW_FOLDER \
+   --optuna
+   ```
+
+7. **Run the training job using the SLURM scheduler**
+
+   To launch a job, use the `sbatch` command with the relevant training script:
+
+```
+sbatch <path-to-training-bash-script>
+```
 
-1.  **Check the status of the training job**
+8.  **Check the status of the training job**
 
     To do this, we can:
 
-    - Check the SLURM logs: these should be created automatically in the directory from which the `sbatch` command is run.
-    - Run supporting SLURM commands (see [below](#some-useful-slurm-commands)).
-    - Check the MLFlow logs. To do this, first create or activate an existing conda environment with `mlflow` installed, and then run the `mlflow` command from the login node.
+- Check the SLURM logs: these should be created automatically in the directory from which the `sbatch` command is run.
+- Run supporting SLURM commands (see [below](#some-useful-slurm-commands)).
+- Check the MLFlow logs. To do this, first create or activate an existing conda environment with `mlflow` installed, and then run the `mlflow` command from the login node.
 
-      - Create and activate a conda environment.
-        ```
-        module load miniconda
-        conda create -n mlflow-env python=3.10 mlflow -y
-        conda activate mlflow-env
-        ```
-      - Run `mlflow` to visualise the results logged to the `ml-runs` folder.
+  - Create and activate a conda environment.
+    ```
+    module load miniconda
+    conda create -n mlflow-env python=3.10 mlflow -y
+    conda activate mlflow-env
+    ```
+  - Run `mlflow` to visualise the results logged to the `ml-runs` folder.
 
-        - If using the "scratch" folder:
+    - If using the "scratch" folder:
 
-          ```
-          mlflow ui --backend-store-uri file:////ceph/zoo/users/sminano/ml-runs-all/ml-runs-scratch
-          ```
+      ```
+      mlflow ui --backend-store-uri file:////ceph/zoo/users/sminano/ml-runs-all/ml-runs-scratch
+      ```
 
-        - If using the selected runs folder:
+    - If using the selected runs folder:
 
-          ```
-          mlflow ui --backend-store-uri file:////ceph/zoo/users/sminano/ml-runs-all/ml-runs
-          ```
+      ```
+      mlflow ui --backend-store-uri file:////ceph/zoo/users/sminano/ml-runs-all/ml-runs
+      ```
 
 ### Some useful SLURM commands
 
 To check the status of your jobs in the queue
 
-```
+````
```

Comment:
```suggestion

```
I think it should be 3 ticks rather than 4 - with 4 it messes up the rendered [view](https://github.com/SainsburyWellcomeCentre/crabs-exploration/blob/nikkna/lr-dev/guides/TrainingModelsHPC.md#some-useful-slurm-commands) of this last section.

Reply:
Yes, should be 3! 🤦🏻‍♀️ not sure what happened there